SETTING_UP_A_DEV_ENVIRONMENT.md

```diff
@@ -0,0 +1,98 @@
+# Setting up a dev environment
+
+So you want to set up a dev environment? AWESOME!!! This doc is here to help you get set up as quickly and painlessly as possible so you can start contributing to storybook.
+
+## Prerequisites
+
+Please have the latest versions of the following on your machine
+
+-   node
+-   npm
+-   yarn
+-   create-react-app
+-   @storybook/cli
+
+Yes, we are using `npm` not `yarn` but you still need the latest version of both as there is a bootstrap script that uses both
```

Comment:
@ndelangen should a separate version be written for `release/3.3` branch, which is yarn-only?

Reply:
I could just update it to use yarn as opposed to npm. Would that be preferable? I wasn't sure what the project's stance was going to be on yarn with the new npm5 and lock files, etc.

Reply:
I should have some time tomorrow

Reply:
@Hypnosphi  Please see this PR: https://github.com/storybooks/storybook/pull/1864

It is the same walkthrough, updated to work with version 3.3.0-alpha.0 and yarn